graphics/gegl-devel/Portfile

```diff
@@ -8,9 +8,9 @@ PortGroup           gobject_introspection 1.0
 name                gegl-devel
 conflicts           gegl gegl-0.3
 set git_name        gegl
-set git_commit      9e9095b600d528b9fa4fc538cd189fe54f29b32f
-set git_date        20180418
-version             0.3.35-${git_date}
+set git_commit      f6caafacf3d8c176049de5c6680f8065bcfe9af9 
+set git_date        20180426
+version             0.4.0-${git_date}
 revision            2
```

Comment:
The `revision` line should be deleted because it starts at 0 when increasing the version and that is the default value.